consensus/blocks.go

```diff
@@ -107,13 +100,16 @@ func (s *State) validHeader(b Block) (err error) {
 		return EarlyTimestampErr
 	}
 
-	// Check that the block is not too far in the future.
-	skew := b.Timestamp - Timestamp(time.Now().Unix())
+	// Check that the block is not too far in the future. An external process
+	// will need to be responsible for resubmitting the block once it is no
+	// longer in the future.
+	skew := int(b.Timestamp) - int(Timestamp(time.Now().Unix()))
```

Comment:
redundant cast here; could just be `int(time.Now().Unix())`. Although I don't know why you're casting to `int` anyway; `FutureThreshold` is untyped, so you could just as well use an `int64` (or a `Timestamp`, for that matter).